ilastik/applets/neuralNetwork/nnClassGui.py

```diff
@@ -413,6 +413,21 @@ def validationMenu():
 
         return menus
 
+    @threadRouted
+    def enableLabellingUI(self, enabled: bool = False):
```

Comment:
For consistency with the Qt-style `setThing` methods:
```suggestion
    def setLabelingUIEnabled(self, enabled: bool):
```